helm/default-apps-aws/templates/apps.yaml

```diff
@@ -8,20 +8,32 @@ metadata:
     chart-operator.giantswarm.io/force-helm-upgrade: "{{ .forceUpgrade }}"
   labels:
     {{- include "labels.common" $ | nindent 4 }}
+    {{- if .inCluster }}  
+    app-operator.giantswarm.io/version: 0.0.0
```

Comment:
Why are we doing this rather than using the version? What's the purpose here?

Reply:
because it's an app bundle

Reply:
@AverageMarcus We are using the concept of an app bundle (i.e. a bundle of multiple apps). For bundles to work, they bundle itself needs to be reconciled by the MC app operator that will create the app cr that will be reconciled by the wc app operator. The only way to ensure the mc app operator reconciles the bundle app is to use 0.0.0

Reply:
Why does it need to be `0.0.0` though? That's not obvious at all. The bundle has a version as it's being used in the values file, why can't that be used here?

Reply:
Why do you think it took us a month to make this work on vintage? 😛
It needs version 0.0.0 because the wc app operators cannot create App CRs on the MC since the escalation CVE found in the app platform was fixed, only the MC one can.

App operators use the version label as a selector to know which App CR to reconcile and the MC app operator uses version 0.0.0 while the WC app operators use their own version. Here is the doc about it https://docs.giantswarm.io/app-platform/app-bundle/#configuring-the-bundles-app-cr-for-the-unique-app-operator